src/component/feature/FeatureToggleList/FeatureToggleListTable/FeatureToggleListTable.tsx

```diff
@@ -141,6 +155,24 @@ export const FeatureToggleListTable: VFC<IExperimentProps> = ({
         }
     }, [setHiddenColumns, isSmallScreen, isMediumScreen]);
 
+    useEffect(() => {
+        const tableState: IPageQuery = {};
+        if (sortBy[0]?.id !== defaultSort.id) {
+            tableState.sort = sortBy[0].id;
+        }
+        if (
+            sortBy[0].desc &&
+            !(sortBy[0]?.id === defaultSort.id && defaultSort.desc)
+        ) {
+            tableState.order = 'desc';
+        }
+        if (globalFilter) {
+            tableState.q = globalFilter;
+        }
+
+        setQuery(tableState);
+    }, [sortBy, globalFilter]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Do we need the `// eslint-disable-line react-hooks/exhaustive-deps`? 

Would be nice to fix stability issues instead of ignoring these warnings.